src/io/flutter/inspector/InspectorTree.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.inspector;
+
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.actionSystem.DataKey;
+import com.intellij.openapi.actionSystem.DataProvider;
+import com.intellij.openapi.actionSystem.PlatformDataKeys;
+import com.intellij.ui.Gray;
+import com.intellij.ui.JBColor;
+import com.intellij.ui.treeStructure.Tree;
+import com.intellij.util.ui.JBUI;
+import com.intellij.xdebugger.impl.ui.DebuggerUIUtil;
+import com.intellij.xdebugger.impl.ui.tree.nodes.XValueNodeImpl;
+import io.flutter.view.InspectorTreeUI;
+import org.jetbrains.annotations.Nullable;
+
+import javax.swing.plaf.TreeUI;
+import javax.swing.plaf.basic.BasicTreeUI;
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreeSelectionModel;
+import java.awt.*;
+
+public class InspectorTree extends Tree implements DataProvider, Disposable {
```

Comment:
This file is partly duplicated with code inside InspectorPanel. Only new delta is the code related to VERY_LIGHT_GREY, scrolling, and highlight root.